system/Test/FeatureTestCase.php

```diff
@@ -289,16 +289,15 @@ public function options(string $path, array $params = null)
 	 *
 	 * @param string      $method
 	 * @param string|null $path
-	 * @param array|null  $params
 	 *
 	 * @return \CodeIgniter\HTTP\IncomingRequest
 	 */
-	protected function setupRequest(string $method, string $path = null, array $params = null): IncomingRequest
+	protected function setupRequest(string $method, string $path = null): IncomingRequest
 	{
 		$config = config(App::class);
 		$uri    = new URI($config->baseURL . '/' . trim($path, '/ '));
 
-		$request      = new IncomingRequest($config, clone($uri), $params, new UserAgent());
+		$request      = new IncomingRequest($config, clone($uri), null, new UserAgent());
```

Comment:
How can I send body?
ex) featureTest
// GET /test-url?type=a
$this->get('/test-url', ['type' => 'a']);
or
// POST /test-url
// request body (type=a)
$this->post('/test-url' ['type => 'a']);

Reply:
Those functions still work like they always did. The $params array wasn't doing anything when sent to IncomingRequest in the constructor anyway. It's handled via a different method.

Reply:
How can I send JSON body?